mlflow/server/handlers.py

```diff
@@ -1103,6 +1104,134 @@ def _default_history_bulk_impl():
     }
 
 
+@catch_mlflow_exception
+@_disable_if_artifacts_only
+def get_metric_history_bulk_interval_handler():
+    MAX_RUNS_GET_METRIC_HISTORY_BULK = 100
+    MAX_RESULTS_PER_RUN = 2500
+    MAX_RESULTS_GET_METRIC_HISTORY = 25000
+
+    args = request.args
+    run_ids = args.to_dict(flat=False).get("run_ids", [])
+    if not run_ids:
+        raise MlflowException.invalid_parameter_value(
+            "GetMetricHistoryBulkInterval request must specify at least one run_id."
+        )
```

Comment:
This is an interesting requirement. Can we just return an empty array?

Reply:
Databricks side enforce run_ids to be non empty

Reply:
Got it :) If Databricks enforces, OSS does the same.

Reply:
Ah ok then! I'll follow Databricks :)